crm/main.py

```diff
@@ -30,7 +30,8 @@ def show_summary(self):
 
         crm_info = configdb.get_entry('CRM', 'Config')
 
-        print '\nPolling Interval: ' + crm_info['polling_interval'] + ' second(s)\n'
+        if crm_info:
+            print '\nPolling Interval: ' + crm_info['polling_interval'] + ' second(s)\n'
```

Comment:
We would need to print some failure message if crm_info is _null_ for some reason!